conf/standard.config

```diff
@@ -0,0 +1,73 @@
+process {
+    withName:
+        "EXPORT_RAW_GENOMES" {
+            container = "ubuntu:20.04"
+            cpus = 1
+            memory = "2 GB"
+    }
+
+    withName:
+        "FASTQC*" {
+            container = 'quay.io/biocontainers/fastqc:0.11.9--0'
+            cpus = 1
+            memory = "2 GB"
+    }
+
+    withName:
+        "MULTIQC*" {
+            container = 'quay.io/biocontainers/multiqc:1.9--pyh9f0ad1d_0'
+            cpus = 1
+            memory = "2 GB"
+    }
+
+    withName:
+        "TRIMMOMATIC"  {
+            container = 'quay.io/biocontainers/trimmomatic:0.35--6'
+            cpus = 1
+            memory = "2 GB"
+    }
+
+    withName:
+        "SPADES" {
+            container = 'quay.io/biocontainers/spades:3.14.0--h2d02072_0'
+            cpus = 1
+            memory = "2 GB"
```

Comment:
@Mxrcon , are you sure that spades only required `1` CPU and `2 GB` RAM?

In my experience, it requires atleast `8 CPU` and `16 RAM`

Reply:
Yes, you're right :smirk: I need to update the cpu/memory part before the final review!